Packs/Slack/Scripts/SlackAskV2/README.md

```diff
@@ -0,0 +1,102 @@
+<p>
```

Comment:
BTW, why is this on HTML and nor markdown?

Reply:
I used the sdk command `cp -r ~/SlackAsk/. ~/SlackAskV2`. I suppose I can use `generate-docs` instead.